core/modules/widgets/action-navigate.js

```diff
@@ -56,7 +56,7 @@ Invoke the action associated with this widget
 */
 NavigateWidget.prototype.invokeAction = function(triggeringWidget,event) {
 	var bounds = triggeringWidget && triggeringWidget.getBoundingClientRect && triggeringWidget.getBoundingClientRect(),
-		suppressNavigation = event.metaKey || event.ctrlKey || (event.button === 1);
+		suppressNavigation = event !== undefined ? event.metaKey || event.ctrlKey || (event.button === 1) : false;
```

Comment:
Here we're really protecting against a missing parameter value, and so we can use a construction that we use elsewhere:

```
NavigateWidget.prototype.invokeAction = function(triggeringWidget,event) {
event = event || {};
```

Reply:
ok :+1: